### ISSUE TITLE
Button component - critical checklist & audit

### ISSUE THREAD

**Issue by amycole501** [uswds/uswds#5425]
Audit the button component for WCAG compliance and critical checklist checks

[Critical Checklist spreadsheet ](https://docs.google.com/spreadsheets/d/1fAJaObQHv7aIaeUpH4fTLmauhmugMtNCrbpZe4BJcPY/edit?usp=sharing)

This ticket will be considered done once the following content has been written and associated WCAG checks are in place for the button component:

1. WCAG criteria related to [button component](https://designsystem.digital.gov/components/button/) elements
2.  What USWDS has done to ensure the current version of the accordion is accessible
3. Best practices for integrating accessibility into the button component
4. What USWDS is unable to test
5. How to perform manual testing (critical checklists) and related user-focused questions for downstream teams to test.
      a. Zoom mag to 200%
     b. Keyboard
     c. JAWS 

The results will be added into the spreadsheet. You can use this document as a starting reference: https://docs.google.com/document/d/1l8KP9A77rg6IF7YME0CvadMMczZKU5FMHvBqkyWFIfM/edit?usp=sharing

**Comment by cory-churches** [comment]
@alex-hull @amycole501  is this likely to be completed before Monday? If not, I can move to the next sprint

**Comment by alex-hull** [comment]
Yes. It has been completed on my end. @cory-churches @amycole501

**Comment by brunerae** [comment]
Closing as complete.
